Backend/sprintone/src/main/java/com/project/omega/bean/dao/JwtResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.project.omega.bean.dao;
+
+import java.io.Serializable;
+
+public class JwtResponse implements Serializable {
+
+    private static final long serialVersionUID = -8091879091924046844L;
+    private final String jwtToken;
+

```

Comment:
JwtResponse class should also return a Date variable which signifies the date and time of the generated token. The returned value should always be enclosed in the toString() method.

Reply:
Why would this be necessary? Expiry date is more useful, a day I think? @HarryPi can you confirm?

Reply:
Yes we agreed the expiration duration would be 24 hours.